### ISSUE TITLE
Speed tooltip sometimes wrong

### ISSUE THREAD

**Issue by KufrA** [Anonomit/ZeraTooltip#3]
![bild](https://github.com/Anonomit/ZeraTooltip/assets/26123598/91dbb866-63dc-4e52-bbc7-873d2bc18786)

Not really sure i found a solid way to reproduce. But if i mouseover the two items they sometimes show the speed of the first weapon. And then if you press "shift" to compare weapons, it will show one speed for both of them.

The speed from the first "speed" option, the one that is next to the DPS is the one that is wrong. 
If i add speed to the left of the "speed meter" (same row as the DPS is on) that speed will be correct (see picture).

Hopefully this second analysis will help you narrow down the bug. I tried to disable ZeraTooltip and the issue is no longer present. So I'm suspecting it has to do with ZeraTooltip, but did not do extensive AB testing.

Playing: WOW Season of discovery 1.15
Interface version (build info): 11500

Otherwise, very neat addon, keep it up =)

**Comment by Anonomit** [comment]
Thanks for the detailed report! I've fixed this in 1a8588b22ebfbd8ee19967e4dbb8c9936273ed7d.

The issue was due to text caching. Since the addon noticed that the weapon damage is the same as what it's seen previously, it skipped figuring out how to rewrite the line and just went with the same thing it did last time. That included the right side text too, which was *not* the same on this weapon.

It'll be fixed in the next version!
